### PR TITLE
Fix the type of component mapping: MapperFunction.

### PR DESCRIPTION
**Pull request by niekraaijmakers**
Fixes the bug : https://github.com/adobe/aem-react-editable-components/issues/47

### CODE REVIEW

**Review by CezCz — APPROVED** [review]

**Comment by sharanyavinod** [comment]
@niekraaijmakers There are some failing checks. Could you please take a look?

**Comment by niekraaijmakers** [comment]
Sorry to say but I am too busy with current projects and personal life. 

I think this is a build error (upload sonarcube report), somebody else could take this on.

**Review by sharanyavinod — APPROVED** [review]

**Comment by sharanyavinod** [comment]
@niekraaijmakers The issue has been resolved in the master branch. Could you please merge the latest changes to the PR?

**Comment by habansal** [comment]
@niekraaijmakers Reminder to merge latest master branch into your private branch to check whether the build passes.
Since, we can't push to your forked repo, so otherwise, we would need to create a new PR for the same.

**Comment by niekraaijmakers** [comment]
done, let's see if it builds.

File: package.json [hunk 1]
```diff
@@ -39,48 +39,48 @@
...
-    "@adobe/aem-spa-page-model-manager": "~1.3.6"
+    "@adobe/aem-spa-page-model-manager": "~1.3.10",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
   },
   "devDependencies": {
-    "@adobe/aem-spa-component-mapping": "~1.1.0",
-    "@adobe/aem-spa-page-model-manager": "~1.3.6",
-    "@adobe/eslint-config-editorxp": "^1.0.5",
+    "@adobe/aem-spa-component-mapping": "file:../aem-spa-component-mapping",
```
Comment (grubyak) [inline package.json:48]: will be updated, once https://github.com/adobe/aem-spa-component-mapping/pull/39/files is merged

**Review by CezCz — APPROVED** [review]

**Comment by grubyak** [comment]
fixed via https://github.com/adobe/aem-react-editable-components/pull/55

**Pull request closed without merging**